docs/guides/mongodb_cluster_okta_idp.md

```diff
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     cyral = {
       source  = "cyralinc/cyral"
-      version = ">= 2.7.0"
+      version = ">= 3.0.0"
```

Comment:
Let's keep it from upgrading to `4.x` in the future and just keep `3.x` updates automatic just in case the user that copies this code is not well-aware of all nuances of this `version` declaration.

```suggestion
      version = "~> 3.0"
```